commit 50de2c9bbbc25074f022b4b2cf9d49f8e9a53e01
Author:     Andreas Schneider <asn@samba.org>
AuthorDate: Tue Oct 2 15:51:08 2012 +0200
Commit:     Andreas Schneider <asn@cryptomilk.org>
CommitDate: Thu Oct 4 12:06:29 2012 +0200

    s3fs-smbd: Make sure the registry is set up before we init printing.
    
    Autobuild-User(master): Andreas Schneider <asn@cryptomilk.org>
    Autobuild-Date(master): Thu Oct  4 12:06:29 CEST 2012 on sn-devel-104
---
 source3/smbd/server.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/source3/smbd/server.c b/source3/smbd/server.c
index 90bbb62..7dad13b 100644
--- a/source3/smbd/server.c
+++ b/source3/smbd/server.c
@@ -1471,6 +1471,10 @@ extern void build_options(bool screen);
 		}
 	}
 
+	if (!dcesrv_ep_setup(ev_ctx, msg_ctx)) {
+		exit(1);
+	}
+
 	/* only start other daemons if we are running as a daemon
 	 * -- bad things will happen if smbd is launched via inetd
 	 *  and we fork a copy of ourselves here */
@@ -1495,10 +1499,6 @@ extern void build_options(bool screen);
 		}
 	}
 
-	if (!dcesrv_ep_setup(ev_ctx, msg_ctx)) {
-		exit(1);
-	}
-
 	if (!is_daemon) {
 		int sock;
 
